3730/2/commands/check_out.rb

```diff
@@ -0,0 +1,34 @@
+module CheckOutCommand
+  include PathCreater
+
+  def checkout!(*)
+    if registred?
+      leave_camp_time
+      save_context :checkout_photo
+      respond_with :message, text: 'Send a selfie'
+    else
+      respond_with :message, text: 'You did not authorize'
+    end
+  end
+
+  def checkout_photo(*)
+    if correct_photo?
+      save_photo(checkout_dir)
+      save_context :checkout_geo
+      respond_with :message, text: 'Send your geoposition'
+    else
+      save_context :checkout_photo
+      respond_with :message, text: 'Please send a selfie!'
+    end
+  end
+
+  def checkout_geo(*)
```

Comment:
Let's have more clear naming. It can happen that somebody will not know what is `geo`